src/polls/PollForm.vue

```diff
@@ -109,26 +149,52 @@
         @click="cancelEditing"
       />
     </div>
+
+    <div v-if="isEditing" class="buttons-container">
```

Comment:
Why do you need two buttons containers? Can we leave only one buttons container but adjust the logic for different cases?